packages/scratch-vm/src/typescript-support/types.d.ts

```diff
@@ -0,0 +1 @@
+export const __esModule: boolean;
```

Comment:
I believe this can be deleted.? Were you finding it necessary for any reason (maybe webpack giving you problems if it wasn't there or something?). 

I'll try deleting it on my end this week and see if it works.